packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -1044,20 +1052,35 @@ class VirtualListBasic extends Component {
 
 	applyStyleToNewNode = (index, ...rest) => {
 		const
-			{itemProps, itemRenderer, getComponentProps} = this.props,
-			key = index % this.state.numOfItems,
-			componentProps = getComponentProps && getComponentProps(index) || {};
+			{itemRefs, itemProps, itemRenderer, getComponentProps} = this.props,
+			{numOfItems} = this.state,
+			key = index % numOfItems,
+			componentProps = getComponentProps && getComponentProps(index) || {},
+			itemContainerRef = (ref) => {
+				if (itemRefs) {
+					itemRefs.current[index % numOfItems] = (ref === null) ? null : ref;
```

Comment:
Could we replace the `(ref === null) ? null : ref;` to 'ref ? ref : null` ?

Reply:
Isn't `index % numOfItems` `key`?

Reply:
It looks the same as just `ref`.

Reply:
> Could we replace the `(ref === null) ? null : ref;` to `ref ? ref : null` ?

Is `ref || null` fine?

Reply:
yes

Reply:
> Isn't `index % numOfItems` `key`?

Yes.... I replaced that part with the key.

Reply:
> It looks the same as just `ref`.

OK.